From: Martin Konrad <konrad@frib.msu.edu>
Date: Mon, 15 Jul 2019 11:15:40 -0400
Subject: Adjust RELEASE file for Debian

---
 configure/RELEASE | 20 +-------------------
 1 file changed, 1 insertion(+), 19 deletions(-)

diff --git a/configure/RELEASE b/configure/RELEASE
index 1f47151..cddaf93 100644
--- a/configure/RELEASE
+++ b/configure/RELEASE
@@ -4,25 +4,7 @@
 
 #NOTE: MSI moved to CONFIG_SITE.
 
-# SNCSEQ required only if testIocStatsApp will be built.
-MAKE_TEST_IOC_APP=YES
-
-# =============================================================
-# Define the version of modules needed by
-# IOC apps or other Support apps - used by testIocStatsApp
-# =============================================================
-SUPPORT=/usr/local/epics/R3.14.11/modules/soft/synApps_5_5/support
--include $(TOP)/../configure/SUPPORT.$(EPICS_HOST_ARCH)
-
-SNCSEQ     = $(SUPPORT)/seq-2-0-12
-#SNCSEQ = /afs/slac/g/spear/epics/modules/seq/seq
-#SNCSEQ = /afs/slac/g/lcls/epics/R3-15-1_1-0/modules/seq/seq-R2-1-17_1-0
-#SNCSEQ = /afs/slac/g/lcls/epics/R3-16-0/modules/seq/seq-2.2.3
-
-#EPICS_BASE = /usr/local/epics/R3.14.11/base
-EPICS_BASE=/afs/slac/g/spear/epics/base
-#EPICS_BASE=/afs/slac/g/lcls/epics/R3-15-1_1-0/base/base-R3-15-1_1-0
-#EPICS_BASE=/afs/slac/g/lcls/epics/R3-16-0/base/base-R3-16-0
+EPICS_BASE=/usr/lib/epics
 
 -include $(TOP)/../configure/EPICS_BASE.$(EPICS_HOST_ARCH)
 
